plugins/inputs/netdata/README.md

```diff
@@ -0,0 +1,64 @@
+# Netdata Input Plugin
+
+The [Netdata](https://github.com/firehol/netdata) plugin reads all metrics
+generated by a Netdata instance.
+
+### Configuration
+
+```toml
+  [[inputs.netdata]]
+  ## Number of points to gather.
+  ## By default inputs plugin run every 10 seconds, by asking 10 points
+  ## to netdata we get measurement for each second. This value is capped by "update every" config of netdata.
+  ## For example you can set
+  ## points = 20
+  ## interval = "10m"
+  ## to get measurement every 30 seconds (and polling netdata every 10 minutes)
+  ## default to 10
+  points = 10
```

Comment:
I guess that's hard to understand for most people. A more natural setting would be the _sampling rate_ or _frequency_ i.e. you specify that you want data with a distance of e.g. `30s`. Do you think it is possible?
If not, please state clearly how to calculate the number of points from a given _interval_ and targeted _frequency_ (even though it is obvious :-)).

Reply:
Mandatory.